### PR TITLE
	 Restructure the running of powershell commands

### PR DESCRIPTION
**Pull request by stefan-caraiman**
Adds a few other arguments required for properly running
powershell commands or scripts.
It also moves the retry_count and retry_delay to the config, to make
it easier for customization.
And also fixes an exception message for when commands timeout.

### CODE REVIEW

File: argus/util.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 import six
 
 
-RETRY_COUNT = 15
+RETRY_COUNT = 7
```
Comment (alexcoman) [inline argus/util.py]: Why isn't this constant a config option ?

**Review by alexcoman — CHANGES_REQUESTED** [review]

File: argus/util.py [hunk 2]
```diff
@@ -246,7 +243,7 @@ def _get_command_powershell(command):
 
 def _get_command_powershell_script(command):
     """Return a valid CMD command that runs a powershell script."""
-    return "powershell -File {}".format(command)
+    return "powershell -Noninteractive -NoLogo -File {}".format(command)
```
Comment (ader1990) [inline argus/util.py]: typo NonInteractive*

**Review by alexcoman — APPROVED** [review]
lgtm

File: argus/config/ci.py [hunk 3]
```diff
@@ -75,6 +75,12 @@ def __init__(self, config):
             cfg.IntOpt("io_upper_timeout", default=IO_UPPER_TIMEOUT,
                        help="Upper timeout for each command that reads or "
                             "writes to a file in the remote instance."),
+            cfg.IntOpt("retry_count", default=7,
+                       help="The retry count that will be run in "
+                            "case a command is failing"),
+            cfg.IntOpt("retry_delay", default=10,
+                       help="The number of seconds that the CI will wait"
```
Comment (ader1990) [inline argus/config/ci.py]: missing space

File: argus/config/ci.py [hunk 4]
```diff
@@ -75,6 +75,12 @@ def __init__(self, config):
             cfg.IntOpt("io_upper_timeout", default=IO_UPPER_TIMEOUT,
                        help="Upper timeout for each command that reads or "
                             "writes to a file in the remote instance."),
+            cfg.IntOpt("retry_count", default=7,
+                       help="The retry count that will be run in "
```
Comment (ader1990) [inline argus/config/ci.py]: The number of retry counts for a failing command

File: argus/config/ci.py (hunk 3 above)
Comment (ader1990) [reply argus/config/ci.py]: The number of seconds between the retries of a failed command

**Review by ader1990 — CHANGES_REQUESTED** [review]

File: argus/config/ci.py [hunk 5]
```diff
@@ -75,6 +75,11 @@ def __init__(self, config):
             cfg.IntOpt("io_upper_timeout", default=IO_UPPER_TIMEOUT,
                        help="Upper timeout for each command that reads or "
                             "writes to a file in the remote instance."),
+            cfg.IntOpt("retry_count", default=15,
+                       help="The retry counts that for a failing command."),
```
Comment (ader1990) [inline argus/config/ci.py]: remove that*

**Pull request merged**